src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -1,50 +1,149 @@
-        <div class="card-content">
-            <app-selectphase [phases]="filteredPhases" [phaseSelected]="phaseSelected()"></app-selectphase>
-        	<div class="phase-submissions" *ngIf="selectedPhase" >
-        		<div>
-              <span class="heading">Today's Remaining Submissions: </span>
-              <span class="content">{{selectedPhaseSubmissions['remaining_submissions_today_count']}}</span>
-            </div>
-            <div>
-              <span class="heading">Total Remaining Submissions: </span>
-              <span class="content">{{selectedPhaseSubmissions['remaining_submissions']}}</span>
+
+      <div *ngIf="isChallengeHost" class="col-md-1 col-sm-1 col-lr-pad">
+        <a class="pointer" (click)="editSubmissionGuideline()">
+          <i class="fa fa-pencil" aria-hidden="true">
+          </i>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="ev-card-panel">
+    <div class="bottom-card-container bottom-hr-line">
+      <strong class="fw-semibold content">
+        Please fill the following fields to submit
+      </strong>
+    </div>
+    <div class="row">
+      <div class="col-md-7 col-sm-7 col-xs-12">
+        <div class="exist-team-card">
+          <ul>
+            <b class="fw-bold content fs-14">Select phase:</b>
+            <app-selectphase [phases]="filteredPhases" [phaseSelectionType]="phaseSelectionType"
+              [phaseSelected]="phaseSelected()"></app-selectphase>
+
+            <div class="form">
+              <div class="row row-lr-margin">
+                <div class="col-md-10 col-sm-10 col-lr-pad submit-upload-file">
+                  <app-input (change)="validateInput($event.target.value)"
+                    [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv, .gz'"
+                    [placeholder]="''" [type]="'file'" [value]="''"
+                    #formsubmit></app-input>
+                </div>
+              </div>
+
+              <div class="row row-lr-margin">
+                <div class="align-left col-sm-10 col-lr-pad">
+                  <div class="input-field align-left">
+                    <app-input [label]="'method_name'" [placeholder]="'Method name (Optional)'"
+                    [type]="'text'" [icon]="" [value]="''"
+                      #formsubmit></app-input>
+                  </div>
+                </div>
+              </div>
+
+              <div class="row row-lr-margin">
+                <div class="align-left col-sm-10 col-lr-pad">
+                  <app-input [label]="'method_description'" [type]="'textarea'"
+                    [placeholder]="'Method description (Optional)'" #formsubmit></app-input><br>
+                </div>
+              </div>
+
+              <div class="row row-lr-margin">
+                <div class="align-left col-sm-10 col-lr-pad">
+                  <app-input [label]="'project_url'" [type]="'text'"
+                    [placeholder]="'Project URL (Optional)'" #formsubmit></app-input>
```

Comment:
Should we have a Regex check for URL's?

Reply:
as there was no check in our current version so I haven't put there. @RishabhJain2018 what do you say about it?

Reply:
yes, a check for the URLs would be good.